src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -433,10 +433,9 @@ public class Options
             /// </summary>
             [Argument(ArgumentType.AtMostOnce,
                 HelpText =
-                    "Whether the input may include quoted values, which can contain separator characters, colons," +
-                    " and distinguish empty values from missing values. When true, consecutive separators denote a" +
-                    " missing value and an empty value is denoted by \"\". When false, consecutive separators" +
-                    " denote an empty value.",
```

Comment:
For the record: the text I removed on this internal doc here was describing a behavior that is no longer true, and that actually mislead me.

I haven't found the PR where the functionality here described was removed, but Yael did told me that in the past the TLC/ML.NET type system did allow for distinguishing "missing" vs "empty" strings, which is no longer true.

This piece of doc I'm removing here was actually removed from the public docs for this same reason on:
https://github.com/dotnet/machinelearning/pull/4899#discussion_r386948608